.vscode/settings.json

```diff
@@ -0,0 +1,63 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "editor.detectIndentation": true,
+  "editor.fontFamily": "'Dank Mono', Menlo, Monaco, 'Courier New', monospace",
```

Comment:
Not a fan of choosing my font for me. A lot of the settings in here are too personal for each developer. How would you re-organize? Or just gitignore this file?

Reply:
ya ya totally agreed... my bad... i had planned to remove it... totally forgot...

Reply:
why is this resolved? I still see the font spec in the settings.json. 

The more I think about it, better to just remove this file completely. We can put an example in some documentation, but I don't want to force editor settings on people. None of the other projects I work on do this.

Could you please remove this file from the PR? Thanks.